Sources/SwiftToolchain/Toolchain.swift

```diff
@@ -118,8 +121,30 @@ public final class Toolchain {
     }
   }
 
+  private func inferManifestDirectory() throws -> AbsolutePath {
+    guard package != nil, var cwd = fileSystem.currentWorkingDirectory else {
+      throw ToolchainError.missingPackage
+    }
+
+    // FIXME: this won't work on Windows
```

Comment:
<!--
  1 failure:  FIXMEs should be ...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: Sources/SwiftToolchain/Toolchain.swift;
  Line: 129;
-->
- :no_entry_sign: FIXMEs should be resolved (this won't work on Windows). (`todo`)